test/test_torch.py

```diff
@@ -9581,6 +9581,44 @@ def test_lerp(self, device):
                 expected = start + weight * (end - start)
                 self.assertEqual(expected, actual)
 
+    def _test_logaddexp(self, device, dtype, base2=False):
```

Comment:
Nit: don't include a default value for base2 (default values are appropriate for user APIs, but this is a developer API). If you want to improve readability you can also make base2 a kwarg-only argument:

```
_test_logaddexp(self, device, dtype, *, base2):
```